guest_c/main.c

```diff
@@ -211,7 +211,7 @@ exports_wasi_sensor_interface_main()
 
         fprintf(stderr, "C guest started\n");
 
-        const char *sensor = "dummy";
+        const char *sensor = "dummy:dummy";
```

Comment:
You mean we can specify
dummy:image
dummy:depth
dummy:imu

if we create dummy image device, dummy depth device, dummy imu device ?

Reply:
Do you think it makes sense ?

`nokhwa:dev/video0 `

Reply:
> You mean we can specify dummy:image dummy:depth dummy:imu

the dummy device ignores the latter part for now.

> if we create dummy image device, dummy depth device, dummy imu device ?

it's possible.

Reply:
> Do you think it makes sense ?
> 
> `nokhwa:/dev/video0 `

sort of. except that /dev/video0 seems like a very linux specific identifier.

Reply:
understood. I agree as the spec.